package.json

```diff
@@ -49,6 +49,7 @@
     "pnpm": {
         "overrides": {
             "ansi-regex": "^5.0.1",
+            "klaw": "^3.0.0",
```

Comment:
What's this again?

Reply:
I have dependency to `@ui5/porject` which has a dependency to `klaw`. `klaw@4` dropped support for Node 12. Without this I am getting an error when doing `pnpm install`.

![image](https://user-images.githubusercontent.com/91968887/155303399-8cb40fbc-738b-48b5-81db-fe560cc48fd1.png)

Is there a better way to do it?

Reply:
@ullasholla, @zdravko-georgiev: I don't see another solution for now. If `@ui5/project` works fine with the`klaw@3` override, we should be fine. We plan to drop support for node12 when SBAS upgrades to node14 soon. We can remove this override then.

Reply:
Thanks @kranthie-sap

Reply:
Could you please also add a note to https://github.com/SAP/open-ux-tools/blob/feat/ui5-proxy-middleware/docs/version-overrides.md and explain why the override is required. You can add there that the override can be removed once we drop support for node12

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/249/commits/c6cbb15e91e080c50227966a931f6b1de661fd91

Reply:
Minor grammatical error. `Newer packages of klaw do not ...` (not does)

Reply:
@zdravko-georgiev, this is now resolved, thanks.